core/Tracker/Response.php

```diff
@@ -78,7 +78,7 @@ public function outputResponse(Tracker $tracker)
                 Common::sendResponseCode(400);
             }
             Common::printDebug("Empty request => Piwik page");
-            echo "<a href='/'>Piwik</a> is a free/libre web <a href='http://piwik.org'>analytics</a> that lets you keep control of your data.";
+            echo 'This resource is part of <a href="https://piwik.org/" rel="nofollow">Piwik</a>, a free/libre web analytics tool that lets you keep control of your data.';
```

Comment:
I think @mattab will want to have the link without the `rel="nofollow"`. Otherwise looks good to me.